virttest/qemu_vm.py

```diff
@@ -1100,6 +1100,7 @@ def add_memorys(devices, params):
                 if params.get("hugepage_path"):
                     backend_options["backend_mem"] = "memory-backend-file"
                     backend_options["mem-path_mem"] = params["hugepage_path"]
+                    backend_options["prealloc_mem"] = params.get("prealloc_mem", "yes")
```

Comment:
```suggestion
                    backend_options["prealloc_mem"] = params.get("vm_mem_prealloc", "yes")
```
to get a consistent naming convention for the VM-level memory object.

Reply:
@luckyh 
Just one question, should this naming convention apply to other places, i.e. tp-qemu, where this param is being requested as `prealloc_mem` ?

Reply:
> @luckyh Just one question, should this naming convention apply to other places, i.e. tp-qemu, where this param is being requested as `prealloc_mem` ?

@mcasquer not really, that's because we want to distinguish the VM-level param (`vm_mem_prealloc`) from the device-level param (`prealloc_mem`) at the current stage and let `prealloc_mem` be only handled by [here](https://github.com/avocado-framework/avocado-vt/blob/7d5d9b0851764e03bdeb6393b9b9e00eb51066e0/virttest/qemu_vm.py#L1061).

Reply:
@luckyh got it, clear from my side